### PR TITLE
Minor navbar changes & VSCode workspace settings

### PR DESCRIPTION
**Pull request by JosefBud**
https://github.com/gusta-project/frontend/commit/59133c3fc9ae5f60afa3e1d7770f4ab9fbff707d Added the logo SVG to the navigation bar.

https://github.com/gusta-project/frontend/commit/3700ca1b7861648a2b45b9281c3eb60e8111c799 Removed the extra "Recipes" link in the navigation, this was a mistake I think I accidentally added in the larger navbar branch I merged.

https://github.com/gusta-project/frontend/commit/5fee3804e6398432499e041cdee314a9ca895a58 Added the VSCode workspace settings file that we spoke about in Mattermost.

https://github.com/gusta-project/frontend/commit/758cceae5803fac79332e0231694ea2e7ce64179 Updated test snapshot.

https://github.com/gusta-project/frontend/pull/29/commits/33e274456820fa31b6619ddbcdc50a2426e01cda Removed the extra "Flavors" link in the navigation and updated the test snapshot.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (ayan4m1) [inline .vscode/settings.json]: Would you mind updating the README to remove this section or mention that it is included by default?

File: media/logo.svg [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
```
Comment (ayan4m1) [inline media/logo.svg]: I have an SVG optimizer that might be able to shrink this guy by quite a bit, but I'll investigate that separately.

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -38,14 +38,15 @@ export default class Header extends Component {
         <Row className="navigation-container">
           <Col>
             <Navbar expand="lg">
-              <Navbar.Brand>MixNJuice</Navbar.Brand>
+              <Navbar.Brand className="pt-0">
+                <img src="/media/logo.svg" alt="logo" width="175" />
```
Comment (ayan4m1) [inline src/components/Header/Header.js]: Would you mind using SCSS to set the width?

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -61,7 +62,6 @@ export default class Header extends Component {
                     )}
                     {this.renderNavDropdownItem('/user/settings', 'Settings')}
                   </NavDropdown>
-                  {this.renderNavItem('/flavors', 'Flavors')}
```
Comment (ayan4m1) [inline src/components/Header/Header.js]: 🥇

File: .vscode/settings.json (hunk 1 above)
Comment (JosefBud) [reply .vscode/settings.json]: 👍 Sure! Updated that.

File: src/components/Header/Header.js (hunk 3 above)
Comment (JosefBud) [reply src/components/Header/Header.js]: 👍 Good idea, updated that!

File: src/style.scss [hunk 5]
```diff
@@ -359,3 +359,7 @@ input {
     z-index: 1;
   }
 }
+
+.image--logo {
+  width: 12rem;
```
Comment (ayan4m1) [inline src/style.scss]: Very nice!

**Pull request merged**